packages/react/src/components/Table/Table.tsx

```diff
@@ -0,0 +1,57 @@
+import { ReactElement, CSSProperties } from 'react';
+import '@compassion-gds/css/src/components/Table/table.css';
+
+export interface ColumnCell {
+  title: string;
+  key: string;
+  headerRenderFn?: (key: string | number, title: string) => ReactElement;
+  cellRenderFn?: (key: string, data: string | number | object) => ReactElement;
```

Comment:
Let's omit the `Fn` from the name, it's clear these are functions from the type interface 

```suggestion
  headerRender?: (col: ColumnCell) => ReactElement;
  cellRender?: (col: ColumnCell, data: object) => ReactElement;
```